<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>distancebetweendates</class>
 <widget class="QWidget" name="distancebetweendates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lTitle">
     <property name="text">
      <string>How many days are between two dates</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="leDayFirst"/>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="leYearSecond"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leDaySecond"/>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="leYearFirst"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lFirstDate">
       <property name="text">
        <string>First date</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lSecondDate">
       <property name="text">
        <string>Second date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="leMonthFirst"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="leMonthSecond"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lDay">
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lMonth">
       <property name="text">
        <string>Month</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="lYear">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lNoOfDays">
       <property name="text">
        <string>No. of days:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leNoDays">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbConfirm">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leDayFirst</tabstop>
  <tabstop>leMonthFirst</tabstop>
  <tabstop>leYearFirst</tabstop>
  <tabstop>leDaySecond</tabstop>
  <tabstop>leMonthSecond</tabstop>
  <tabstop>leYearSecond</tabstop>
  <tabstop>leNoDays</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbConfirm</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
